### ISSUE TITLE
 ModuleNotFoundError: No module named 'sinfo'

### ISSUE THREAD

**Issue by AAA-3** [theislab/scvelo#688]
Importing scvelo gives the following error:

It was running fine but has now stopped importing on jupyter notebooks and the terminal. I have `sinfo 0.3.4`  and `scvelo 0.2.4`

I have tried ´conda update --all´ and uninstalling and reinstalling scvelo which has not helped. Any ideas?

```

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_52845/1163986323.py in <module>
      1 #import anndata
----> 2 import scvelo as scv
      3 #import pandas as pd
      4 #import numpy as np
      5 #import matplotlib.pyplot as plt

~/anaconda3/lib/python3.8/site-packages/scvelo/__init__.py in <module>
      1 """scvelo - RNA velocity generalized through dynamical modeling"""
      2 from anndata import AnnData
----> 3 from scanpy import read, read_loom
      4 
      5 from scvelo import datasets, logging, pl, pp, settings, tl, utils

~/anaconda3/lib/python3.8/site-packages/scanpy/__init__.py in <module>
      4 
      5 if not within_flit():  # see function docstring on why this is there
----> 6     from ._utils import check_versions
      7 
      8     check_versions()

~/anaconda3/lib/python3.8/site-packages/scanpy/_utils/__init__.py in <module>
     23 from packaging import version
     24 
---> 25 from .._settings import settings
     26 from .._compat import Literal
     27 from .. import logging as logg

~/anaconda3/lib/python3.8/site-packages/scanpy/_settings.py in <module>
      9 from typing import Tuple, List, ContextManager
     10 
---> 11 from . import logging
     12 from .logging import _set_log_level, _set_log_file, _RootLogger
     13 from ._compat import Literal

~/anaconda3/lib/python3.8/site-packages/scanpy/logging.py in <module>
     10 
     11 import anndata.logging
---> 12 from sinfo import sinfo
     13 
     14 

ModuleNotFoundError: No module named 'sinfo'
```

**Comment by WeilerP** [comment]
Based on the Traceback, this is a Scanpy issue. Please open an issue there.

**Comment by AAA-3** [comment]
Thanks! 
For future if anyone runs into this:

I uninstalled and reinstalled Anndata, scVelo and scanpy and ran conda update --all multiple times to let conda sort out environment inconsistencies. Then I downloaded sinfo again (without uninstalling it) after which conda updated the package. I ran conda update --all again to see if there were any other inconsistencies (there were) and once that was resolved the script ran perfectly fine. 

So not sure if it was scanpy or scvelo...might have been something completely different...Sorry I can't be more insightful.
